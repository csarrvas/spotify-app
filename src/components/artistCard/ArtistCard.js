import React from 'react';

const ArtistCard = ({ artist }) => {
  return (
    <div className="artist-related">
      <p>{artist.name}</p>
      <a target="_blank" rel="noopener noreferrer" href={artist.external_urls.spotify}>
        <i className="fas fa-play-circle">Go to this artist on spotify</i>
      </a>
    </div>
  );
}
 
export default ArtistCard;