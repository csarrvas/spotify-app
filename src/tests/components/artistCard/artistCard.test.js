import React from 'react';
import { shallow } from 'enzyme';
import ArtistCard from '../../../components/artistCard/ArtistCard';
import { artist } from '../../fixtures/data';

test('should render ArtistCard correctly', () => {
  const wrapper = shallow(<ArtistCard artist={artist}/>);
  expect(wrapper).toMatchSnapshot();
});