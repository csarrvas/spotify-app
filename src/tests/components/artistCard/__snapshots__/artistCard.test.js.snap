// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ArtistCard correctly 1`] = `
<div
  className="artist-related"
>
  <p>
    Flo Rida
  </p>
  <a
    href="https://open.spotify.com/artist/0jnsk9HBra6NMjO2oANoPY"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      className="fas fa-play-circle"
    >
      Go to this artist on spotify
    </i>
  </a>
</div>
`;
